releasenotes/notes/circuit-key-supports-controlflow-a956ebd2fcebaece.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Primitives support the dynamic circuits with control flow.
+    If backend supports it, users can run the circuits using :class:`~BackendSampler`
+    and :class:`~BackendEstimator`.
```

Comment:
Does this change apply to only the `BackendSampler` and `BackendEstimator`, or is this a general improvement that affects all `BaseSampler` and `BaseEstimator` subclasses?

Reply:
You are right. When I wrote this document, I thought that if a backend did not support control flow, primitives also did not support, but I was wrong.